### PR TITLE
Re-add barrier-free and organization box to POI

### PR DESCRIPTION
**Pull request by JoeyStk**
### Short description

The organization and barrier-free box in our POI sidebar were removed in #2043.

### Proposed changes

- Include the template for organization and barrier-free to our poi form
- Write tests to make sure they also appear in the sidebar

### Side effects

- The poi_form probably needs some more testing, but I'm not sure if this should be done in this PR

### Resolved issues

Fixes: #2769

__________________________________________________

[Pull Request Review Guidelines](https://digitalfabrik.github.io/integreat-cms/pull-request-review-guide.html)

### CODE REVIEW

File: integreat_cms/cms/templates/pois/poi_form.html [hunk 1]
```diff
@@ -12,8 +12,8 @@
           enctype="multipart/form-data"
           data-unsaved-warning>
         {% csrf_token %}
-        <div class="w-full flex flex-wrap justify-between mb-4">
-            <h1 class="heading overflow-hidden text-ellipsis">
+        <div class="flex flex-wrap justify-between w-full mb-4">
```
Comment (timobrembeck) [inline integreat_cms/cms/templates/pois/poi_form.html:15]: Did you use a tool to sort the classes alphabetical? If so, I'd like to include this into our code style tools :grin:

File: tests/cms/views/poi/test_poi_form.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    response = client.get(edit_poi)
+
+    if role == ANONYMOUS:
+        assert response.status_code == 302
+        assert (
+            response.headers.get("location") == f"{settings.LOGIN_URL}?next={edit_poi}"
+        )
+        return
+
+    assert organization_box in response.content.decode("utf-8")
```
Comment (timobrembeck) [inline tests/cms/views/poi/test_poi_form.py:54]: Hmm, I think those two tests can easily be combined to reduce the testing time.

**Review by timobrembeck — APPROVED** [review]
Fixes the bug, thanks! :+1:

**Review by MizukiTemma — APPROVED** [review]
Appart from the suggestion for the tests, it looks good :)

File: integreat_cms/cms/templates/pois/poi_form.html (hunk 1 above)
Comment (JoeyStk) [reply integreat_cms/cms/templates/pois/poi_form.html:15]: Yes, I did. I use the [Headwind extension](https://marketplace.visualstudio.com/items?itemName=heybourn.headwind) for VS Code :)

**Pull request merged**